### PR TITLE
Tremerent artefact docs

### PR DESCRIPTION
**Pull request by tremerent**
Probably want to test on your machine because there's a fair bit of weird LINQ stuff that may or may not be fully kosher.

Fun hint, since I know you're not totally onboard with my super-fine commits: you can change "Changes from 1 commit" to "View all changes" when viewing the pull request commits, and then it's like it's all just one big happy commit!

### CODE REVIEW

File: ClientApp/src/components/Artefact/ArtefactDocs.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { decycle } from 'json-cyclic';
```
Comment (jsolem) [inline ClientApp/src/components/Artefact/ArtefactDocs.js]: Remove decycle import

File: ClientApp/src/components/UserProfile.js [hunk 2]
```diff
@@ -6,22 +6,6 @@ import { changeCurUserInfo } from '../scripts/requests.js';
 
 import './UserProfile.css';
 
-//function getProfile(userId) {
```
Comment (jsolem) [inline ClientApp/src/components/UserProfile.js]: Personally I would have kept this ....

**Comment by jsolem** [comment]
> Fun hint, since I know you're not totally onboard with my super-fine commits: you can change "Changes from 1 commit" to "View all changes" when viewing the pull request commits, and then it's like it's all just one big happy commit!

I take that back!! I think this granularity is great!! If anything I'm too course. This is perfecto imo

File: ClientApp/src/components/UserProfile.js (hunk 2 above)
Comment (tremerent) [reply ClientApp/src/components/UserProfile.js]: I'm sorry but Granny Bample has asked that her personal information be removed from all Artefactor services and databases. Something about our "total incompetence" and "blatant disregard for people who like family trees".

**Pull request merged**